### PR TITLE
add doc for add-contact-into-room, change roomTopic

### PR DESCRIPTION
**Pull request by YipZong**
add doc for add-contact-into-room, change roomTopic

- [v] Create your contributor profile (if you are a first time contributor)

### CODE REVIEW

**Review by lijiarui — CHANGES_REQUESTED** [review]
Thanks so much for the contribution!

Could you refer to https://github.com/wechaty/wechaty-getting-started to give more general docs for this?

Thanks!

**Comment by YipZong** [comment]
@lijiarui Thanks for the comment. Added some general doc. Please take a look again. Thank you!

File: docusaurus/docs/howto/room.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
 title: 'Managing rooms'
 ---
 
+Managing room is one of the important features in WeChaty. The word "room" here refers to Wechat rooms (also called groups).
```
Comment (lijiarui) [inline docusaurus/docs/howto/room.md]: WeChaty --> Wechaty

File: docusaurus/docs/howto/room.md [hunk 2]
```diff
@@ -131,7 +143,47 @@ class MyBot(Wechaty):
...
-// TODO: Pull Request is welcome!
+import {
+  Contact,
+  Wechaty,
+  log,
+  Room,
+} from 'wechaty'
+
+//helper function
+async function putPersonIntoRoom(person: Contact, room: Room) {
```
Comment (lijiarui) [inline docusaurus/docs/howto/room.md]: According https://github.com/wechaty/wechaty-getting-started/blob/main/examples/advanced/room-bot.js 

Maybe we change to `putInRoom()` is better.

File: docusaurus/docs/howto/room.md [hunk 3]
```diff
@@ -131,7 +143,47 @@ class MyBot(Wechaty):
...
+    await room.add(person);
+  } catch (e) {
+    //any error will be here
+    log.error("Bot", "putPersonIntoRoom() exception: " + e.stack);
+  }
+}
+
+//in the main code
+//If the secrete code is ZhiMaKaiMen
+if (msg.text() === 'ZhiMaKaiMen') {
```
Comment (lijiarui) [inline docusaurus/docs/howto/room.md]: Wechaty always using ding as the test message. So I suggesting changing to ding in official docs.

**Review by lijiarui — CHANGES_REQUESTED** [review]
Thanks for imporving wechaty docs.

Please refer: https://github.com/wechaty/wechaty-getting-started/blob/main/examples/advanced/room-bot.js

And change according to wechaty official example. Then we would merge this!

Thanks again!

File: docusaurus/docs/howto/room.md (hunk 1 above)
Comment (YipZong) [reply docusaurus/docs/howto/room.md]: changed to lower case in the latest commit. Please take a look!

File: docusaurus/docs/howto/room.md (hunk 2 above)
Comment (YipZong) [reply docusaurus/docs/howto/room.md]: Method name changed to `putInRoom` in the latest commit. Please take a look! Thank you！

File: docusaurus/docs/howto/room.md (hunk 3 above)
Comment (YipZong) [reply docusaurus/docs/howto/room.md]: Noted. Changed the secret to ding. Please take a look!

**Review by huan — APPROVED** [review]
LGTM

**Review by lijiarui — APPROVED** [review]

**Comment by lijiarui** [comment]
Thank you very much for your contribution!

## You are welcome to join [Wechaty Contributor Program](https://wechaty.js.org/docs/contributor-program/)

### 1. Join Wechaty Organization

> You've invited YipZong to Wechaty! They'll be receiving an email shortly. They can also visit https://github.com/wechaty to accept the invitation.

I have invited you to join our Wechaty GitHub Organization, please accept it by following the above message.

### 2. Update Your Wechaty Contributor Profile

1. Please open [Contributor Hall of Fame](https://docs.google.com/spreadsheets/d/1XcDoIczyIclqXP1p90Sz7S0n4Q22xjFzJmjPFlU2g1E/edit#gid=0) and add yourself to the end of the list, so that other contributors will know you better!
1. Please add yourself to our Website [Contributors](https://wechaty.js.org/contributors/) by [creating a PR](https://github.com/wechaty/wechaty.js.org/tree/master/jekyll/_contributors) and refer to this PR link as well.

### 3. Join The Contributor Only WeChat Room

We also have a WeChat room for contributors only which can discuss Wechaty at a deeper level, you are welcome to join and if you are interested.

Please add my: wechat: _**ruirui_0914**_  and send this pr link to me. I will invite you into Wechaty Contributor Room

Link to https://github.com/wechaty/PMC/issues/16

Cheers!

**Pull request merged**